### PR TITLE
Nuovo Gioco : Cappuccetto Rosso

### PR DESCRIPTION
**Pull request by AlbertoSalso1974**
New Game Aka Zukin
https://unmamed.mameworld.info/non_other80.html#AkaZukin
Git Hub is hard and we can only edit file in browser. Can you help?
Game plate and ROM provided by Ignacio Seki. Mame emulator code by Alberto Salso
here is the ROM [removed link]

### CODE REVIEW

File: src/mame/orca/vastar.cpp [hunk 1]
```diff
@@ -1,6 +1,10 @@
 // license:BSD-3-Clause
 // copyright-holders: Allard van der Bas
 
+// Game plate and ROM provided by Ignacio Seki. Mame emulator code by Alberto Salso
+// here is the ROM
+// https://www.mediafire.com/file/r0qft5t6z1jpvk6/akazukin.zip/file
```
Comment (angelosa) [inline src/mame/orca/vastar.cpp:6]: No rom links in source file

File: src/mame/orca/vastar.cpp [hunk 2]
```diff
@@ -309,6 +352,9 @@ void vastar_state::draw_sprites(bitmap_ind16 &bitmap, const rectangle &cliprect)
 //  for (int offs = 0; offs < 0x40; offs += 2)
 	for (int offs = 0x40 - 2; offs >= 0; offs -= 2)
 	{
+		if (!(offs & 0x10))
```
Comment (angelosa) [inline src/mame/orca/vastar.cpp:355]: This plus other mods in PR will cause regressions in the other games used by this driver.

File: src/mame/orca/vastar.cpp
Comment (angelosa) [inline src/mame/orca/vastar.cpp:1]: This won't compile without modding [mame.lst](https://github.com/mamedev/mame/blob/master/src/mame/mame.lst) with the new addition.

File: src/mame/orca/vastar.cpp (hunk 2 above)
Comment (DavidHaywood) [reply src/mame/orca/vastar.cpp:355]: I think these changes are correct for the other games.  I haven't tried this PR, but it does look like there are 2 lists of 0x10 entries, with gaps between them.

I suspect Planet Probe might expect them to be drawn swapped (but still in order) too, I noted that when I worked on it some years ago.  Other changes could be correct for the rest of the games too, probably same video system with some lines swapped?

File: src/mame/orca/vastar.cpp (hunk 1 above)
Comment (mikehyrus) [reply src/mame/orca/vastar.cpp:6]: Agreed, best not to expose the ROM’s webpage of origin while others would search via Google or Bing.

**Comment by DavidHaywood** [comment]
> I believe this should really be a stand-alone driver, it barely fits inside `orca/vastar.cpp` and while I see some pattern in the video section it otherwise don't really match to me.

I think the driver to start with is a poor example for external contributors to be honest.  Looking at it now, this addition has more in common than the dogfight clone has, which just seems to be transplanting the content of zodiack.cpp into here; that one doesn't even share the video system with these games.

I can see an argument being made to convert the video system here into a configurable device, and having these as separate drivers, but I also fear we'll overwhelm an external contributor who already stated they're struggling with Github by making such requests.

**Comment by angelosa** [comment]
Well this needs somebody willing to assign himself to the task. As it stands this PR is not usable in current form anyway, so making a PR/commit from scratch is probably the best option.

**Comment by angelosa** [comment]
Addressed my concerns over PR #11014, can close this up.

**Comment by angelosa** [comment]
@AlbertoSalso1974 is it possible to have high res PCB scans?

**Pull request closed without merging**